src/frontend/components/SøknadsSteg/Forside/BekreftelseOgStartSoknad.tsx

```diff
@@ -106,9 +105,7 @@ const BekreftelseOgStartSoknad: React.FC = () => {
                 <BodyShort weight={'semibold'}>
                     <SpråkTekst id="forside.bekreftelsesboks.tittel" />
```

Comment:
Mangler resten av tekstene for bekreftelsesboks, fordi jeg må avklare med @daphneleebeek om vi ønsker å gå bort fra `LocaleRecordString` i sanity (og resten av feltene er av denne typen)